data_visualization_in_ggplot2/data_visualization_ggplot2.md

```diff
@@ -18,7 +22,58 @@ After completion of this module, learners will be able to:
 
 @end
 
-link: https://chop-dbhi-arcus-education-website-assets.s3.amazonaws.com/css/modules.css
+@gifPreload
+<script>
+(function($) {
+
+  // Get the .gif images from the "data-alt".
+	var getGif = function() {
+		var gif = [];
+		$('img').each(function() {
+			var data = $(this).data('alt');
+			gif.push(data);
+		});
+		return gif;
+	}
+
+	var gif = getGif();
+
+	// Preload all the gif images.
+	var image = [];
+
+	$.each(gif, function(index) {
+		image[index]     = new Image();
+		image[index].src = gif[index];
+	});
+
+	// Change the image to .gif when clicked and vice versa.
+	$('figure').on('click', function() {
+
+		var $this   = $(this),
+				$index  = $this.index(),
+
+				$img    = $this.children('img'),
+				$imgSrc = $img.attr('src'),
+				$imgAlt = $img.attr('data-alt'),
+				$imgExt = $imgAlt.split('.');
+
+		if($imgExt[1] === 'gif') {
+			$img.attr('src', $img.data('alt')).attr('data-alt', $imgSrc);
+		} else {
+			$img.attr('src', $imgAlt).attr('data-alt', $img.data('alt'));
+		}
+
+		// Add play class to help with the styling.
+		$this.toggleClass('play');
+
+	});
+
+})(jQuery);
+</script>
+@end
+
+link:  https://chop-dbhi-arcus-education-website-assets.s3.amazonaws.com/css/styles.css
+script: https://kit.fontawesome.com/83b2343bd4.js
```

Comment:
Also need to add - 
script:  https://code.jquery.com/jquery-3.6.0.slim.min.js

Reply:
Good catch, thank you!